sdk/core/AzureCore/Source/Pipeline/PipelineContext.swift

```diff
@@ -139,6 +159,26 @@ public class PipelineContext {
         }
         return dict
     }
+
+    /// Add a `CancellationToken` while applying smart defaulting logic. If the client transport options
+    /// specify a timeout, this will be used to automatically create `CancellationToken`s for each call,
+    /// even when a token is not specified. If the client call options contain a `CancellationToken` with
+    /// no timeout the default timeout will be applied, if specified in `AzureClientOptions`.
+    /// - Parameters:
+    ///   - cancellationToken: Optional `CancellationToken` object.
+    ///   - clientOptions: Optional `AzureClientOptions`
+    public func add(cancellationToken: CancellationToken?, applying clientOptions: AzureClientOptions?) {
```

Comment:
`clientOptions` is never nil, so we can make this non-optional.

Reply:
Done.